packages/react/src/generators/application/lib/create-application-files.ts

```diff
@@ -4,12 +4,20 @@ import { join } from 'path';
 
 export function createApplicationFiles(host: Tree, options: NormalizedSchema) {
   let styleSolutionSpecificAppFiles: string;
-  if (options.styledModule && options.style !== 'styled-jsx') {
-    styleSolutionSpecificAppFiles = '../files/styled-module';
-  } else if (options.style === 'styled-jsx') {
-    styleSolutionSpecificAppFiles = '../files/styled-jsx';
-  } else if (options.style === 'none') {
-    styleSolutionSpecificAppFiles = '../files/none';
+  if (options.styledModule) {
+    switch (options.style) {
```

Comment:
Logic was getting a bit deluated so moving to a switch statement made it easier for my brain to parse.